subsys/bluetooth/shell/gatt.c

```diff
@@ -1294,7 +1328,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(gatt_cmds,
 	SHELL_CMD_ARG(unregister, NULL,
 		      "unregister pre-predefined test service",
 		      cmd_unregister_test_svc, 1, 0),
-	SHELL_CMD_ARG(notify, NULL, "[data]", cmd_notify, 1, 1),
+	SHELL_CMD_ARG(notify, NULL, "<handle> <conn> <str>", cmd_notify, 3, 0),
```

Comment:
The number of mandatory arguments should be 4 because it include the command name.

Reply:
```suggestion
	SHELL_CMD_ARG(notify, NULL, "<handle> <conn> <str>", cmd_notify, 4, 0),
```